panel/webserver_configuration.md

```diff
@@ -50,3 +50,47 @@ sudo ln -s /etc/apache2/sites-available/pterodactyl.conf /etc/apache2/sites-enab
 sudo a2enmod rewrite
 systemctl restart apache2
 ```
+
+## Caddy
+Caddy is a webserver with automatic https, this may come handy when you don't want to remake your ssl certificates every 90 days.
+::: danger
+Caddy is incompatible with other webservers, if you plan on running other websites on your panel you will have to add them to caddy too.
+:::
+/etc/caddy/Caddyfile:
+``` json
```

Comment:
This isn't JSON, just do the backticks without anything since its basically just curly text.